### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by andytael**
Remove parse and doc updates

### CODE REVIEW

File: docs-source/spring/content/setup/_index.md [hunk 1]
```diff
@@ -116,6 +113,113 @@ You must meet the following prerequisites to use Oracle Backend for Spring Boot
   - Oracle Backend for Spring Boot and Microservices command-line interface (oractl). [Download oractl](https://github.com/oracle/microservices-datadriven/releases).
   - [OPTIONAL]Oracle Backend for Spring Boot and Microservices VS Code Extension. [Download VS Code Extension](https://github.com/oracle/microservices-datadriven/releases).
 
+## OCI policies
+
+The following policies needs to be in place to be able to install Oracle Backend for SPring Boot and Microservices. Top level and their dependencies listed.
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:118]: typo SPring

File: docs-source/spring/content/setup/_index.md [hunk 2]
```diff
@@ -116,6 +113,113 @@ You must meet the following prerequisites to use Oracle Backend for Spring Boot
...
+```
+
+### Vault
+
+When Vault is in Use
+
+```text
+Allow group `<group-name>` to manage vaults in `<location>`
+Allow group `<group-name>` to manage keys in `<location>`
+└── Allow group `<group-name>` to use vaults in `<location>` [^1]
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:181]: did you mean to leave this in?

File: docs-source/spring/content/setup/_index.md [hunk 3]
```diff
@@ -116,6 +113,113 @@ You must meet the following prerequisites to use Oracle Backend for Spring Boot
...
+```text
+Allow group `<group-name>` to read orm-template in `<location>`
+Allow group `<group-name>` to use orm-stacks in `<location>`
+└── Allow group `<group-name>` to manage orm-jobs in `<location>`
+Allow group `<group-name>` to manage orm-private-endpoints in `<location>`
+```
+
+#### Additional Oracle Resource Manager
+
+To allow building ORM Stacks:
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:208]: this is not needed for an end user install is it?

File: docs-source/spring/content/setup/_index.md [hunk 4]
```diff
@@ -116,6 +113,113 @@ You must meet the following prerequisites to use Oracle Backend for Spring Boot
...
+
+```text
+allow group `<group-name>` to manage orm-template in `<group-name>`
+allow group `<group-name>` to manage orm-stacks in `<group-name>`
+```
+
+### Cloud Shell
+
+For workshops:
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:217]: needed?

File: docs-source/spring/content/setup/_index.md (hunk 3 above)
Comment (gotsysdba) [reply docs-source/spring/content/setup/_index.md:208]: Additional ORMs are not required for the installation user; can be removed

File: docs-source/spring/content/setup/_index.md (hunk 4 above)
Comment (gotsysdba) [reply docs-source/spring/content/setup/_index.md:217]: Not needed.

File: docs-source/spring/content/setup/_index.md (hunk 2 above)
Comment (gotsysdba) [reply docs-source/spring/content/setup/_index.md:181]: manage  keys grants use vaults, so can take-out the use vaults and leave the two..

File: docs-source/spring/content/setup/_index.md [hunk 5]
```diff
@@ -173,16 +173,14 @@ Allow group `<group-name>` to manage autonomous-database-family in `<location>`
...
 ### Vault
 
-When Vault is in Use
+If you deploy Oracle Backend for Spring Boot and Microservices **STANDARD** edtion you need the following policies.
 
 ```text
 Allow group `<group-name>` to manage vaults in `<location>`
 Allow group `<group-name>` to manage keys in `<location>`
-└── Allow group `<group-name>` to use vaults in `<location>` [^1]
+└── Allow group `<group-name>` to use vaults in `<location>`
```
Comment (markxnelson) [inline docs-source/spring/content/setup/_index.md:181]: shoudln't this line be removed based on john's confirmation?

**Review by markxnelson — APPROVED** [review]

**Pull request merged**